tests/test_wave_03.py

```diff
@@ -55,12 +55,13 @@ def test_friends_unique_movies_not_duplicated():
     # Assert
     assert len(friends_unique_movies) == 3
 
-    raise Exception("Test needs to be completed.")
+    assert friends_unique_movies.count(INTRIGUE_3) == 1
+    # raise Exception("Test needs to be completed.")
```

Comment:
This is a good test! We could also compare the `friends_unique_movies` list to a list containing the movie dictionaries themselves.